testing/php7-memcached/APKBUILD

```diff
@@ -15,7 +16,7 @@ pecldepends="php7-dev autoconf zlib-dev"
 makedepends="$pecldepends libmemcached-dev"
 install=""
 subpackages=""
-source="$pkgname-$pkgver.tar.gz::https://github.com/$_pkgrepo-dev/$_pkgrepo/archive/$_pkgbranch.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/$_pkgrepo-dev/$_pkgrepo/archive/$_pkgcommit.tar.gz"
 
 builddir="$srcdir"/$_pkgrepo-$_pkgbranch
```

Comment:
shouldn't the builddir also match the commit package?  I think that's why travis builds are failing.  try changing $_pkgbranch to $_pkgcommit on the builddir= line